braindecode/models/deep4.py

```diff
@@ -308,24 +319,27 @@ def add_conv_pool_block(
                 init.constant_(bnorm_weight, 1)
                 init.constant_(bnorm_bias, 0)
 
-        init.xavier_uniform_(self.conv_classifier.weight, gain=1)
-        init.constant_(self.conv_classifier.bias, 0)
+        init.xavier_uniform_(self.final_layer[0].weight, gain=1)
+        init.constant_(self.final_layer[0].bias, 0)
 
         # Start in eval mode
         self.eval()
 
     def load_state_dict(self, state_dict, *args, **kwargs):
         """Wrapper to allow for loading of a state_dict from a model before CombinedConv was
          implemented"""
-        keys_to_change = [
+        keys_time_spat = [
             "conv_time.weight",
             "conv_spat.weight",
             "conv_time.bias",
             "conv_spat.bias",
         ]
-        new_state_dict = OrderedDict()
+        state_dict_time_spat = OrderedDict()
         for k, v in state_dict.items():
-            if k in keys_to_change:
+            if k in keys_time_spat:
                 k = f"conv_time_spat.{k}"
-            new_state_dict[k] = v
+            state_dict_time_spat[k] = v
+
+        # I just added this to dan's function
```

Comment:
I would remove this kind of comments, only code and some important info for the code that could be hard to understand. If you want to let us know something just leave a comment in PR :smiley: 

```suggestion
```